### PR TITLE
feat(releases): Link user to release details page when clicked on the adoption chart

### PR DESCRIPTION
**Pull request by taylangocmen**
This adds a click listener to the release adoption chart, it sends user to the release details page. You need to click very precisely on the line on the chart: the red circled area in the screenshot.

Screen Shot 2021-06-08 at 8 39 32 PM

Resolves [WOR-941](https://getsentry.atlassian.net/browse/WOR-941)

### CODE REVIEW

**Comment by matejminar** [comment]
I'm kind of worried that people won't know that they can click on that tiny little dot.

It would be nice if we could do something similar to this:
https://echarts.apache.org/examples/en/editor.html?c=line-race
Where when you hover over a single line, all others fade out and capture a click then.

File: static/app/views/releases/list/releaseAdoptionChart.tsx [hunk 1]
```diff
@@ -1,83 +1,171 @@
...
+  handleClick = (params: {seriesId: string}) => {
+    const {organization, router, selection} = this.props;
+
+    const project = selection.projects[0];
+
+    router.push({
+      pathname: `/organizations/${organization?.slug}/releases/${encodeURIComponent(
+        params.seriesId
+      )}/`,
+      query: {project},
```
[the remark this answers is not in the document]
Comment (taylangocmen) [reply static/app/views/releases/list/releaseAdoptionChart.tsx:167]: 👍

**Comment by taylangocmen** [comment]
> I'm kind of worried that people won't know that they can click on that tiny little dot.
> 
> It would be nice if we could do something similar to this:
> https://echarts.apache.org/examples/en/editor.html?c=line-race
> Where when you hover over a single line, all others fade out and capture a click then.

Yeah this is very nice, I'll do that

File: static/app/views/releases/list/releaseAdoptionChart.tsx [hunk 2]
```diff
@@ -92,7 +92,7 @@ class ReleaseAdoptionChart extends AsyncComponent<Props, State> {
             interval: getInterval(
               {
                 start: location.query.start,
-                end: location.query.end,
+                end: location.query.start,
```
Comment (matejminar) [inline static/app/views/releases/list/releaseAdoptionChart.tsx:95]:
```suggestion
                end: location.query.end,
```

**Review by matejminar — APPROVED** [review]

**Pull request merged**